docs/customer-portal/cp-did.md

```diff
@@ -5,8 +5,27 @@ A **DID** number is one that exists on the public telephone network. When dialed
 To edit **DID (Direct Inward Dial)**:
 
 1. Select one DID by clicking on the number, or select multiple and then click **`Bulk Edit`**.
+2. For single DID, click on number, click on **Edit** (beside Destination) and you will see the following fields in a new window.
+
     + **URI:** Set the **Destination DID** (number or extension) and **Destination IP** to forward calls to, or a specific Session Initiation Protocol (SIP) Uniform Resource Identifier (URI)  (formatted without the initial `sip:`, for example, `1-999-123-4567@voip-provider.example.net`)
     + **External:** Prefix sends the call to the internet and then the number to send the call to.

```

Comment:
To send the call back out to the internet

Reply:
made the correction